FAQ.md

```diff
@@ -35,7 +37,9 @@
     *   `getting-started`: issue or pull request is suitable for someone new to GitHub and/or this project
     *   `help-wanted`: a question or request for assistance
     *   `left-as-was`: marks an issue closed because the item in question will be left as it was
+    *   `merge`: issue or pull request related to merge changes from "third-party" repository
     *   `suitable-for-newcomer`: issue or pull request is a good starting point for someone who is relatively new to GitHub and/or this project
+    *   `validator`: issue or pull request related with validator
```

Comment:
Not everyone will know what the "validator" refers to... suggested wording:

"involves changes to how lessons are validated for compliance with the lesson template"